bgpd/bgp_mplsvpn.c

```diff
@@ -1487,6 +1487,7 @@ static mpls_label_t bgp_mplsvpn_get_label_for_path_vrf(struct bgp_path_info *pi,
 			LIST_INSERT_HEAD(&(blnc->paths), pi, label_nh_thread);
 			pi->label_nexthop_cache = blnc;
 			pi->label_nexthop_cache->path_count++;
+			blnc->last_update = monotime(NULL);
```

Comment:
you set the value but where is this value used?

Is it for the next commit ? If so, say it in the commit log!

Reply:
```
The timer value will be used for troubleshooting by a show
command in the next commit.
```